### DISCUSSION TITLE
Mobile app not accessible behind NGINX

### DISCUSSION THREAD

**Discussion by micogg** [immich-app/immich#812] [Q&A] [answered]
Hi all,

I am accessing Immich through a custom domain in which is handled by NGINX Proxy Manager.
I can access Immich from the public URL via browser both internally and externally, but the app does not allow me to connect.
I receive a popup stating the username, password or server information is incorrect, but it works with the internal IP.

Is there a way to see a more detailed error log from the app to find why the public IP does not work with the app?

The mobile app is android by the way.

Thanks

**Comment by PixelJonas** [comment, marked as answer]
Can you post your configuration? Are you pointing your NGINX Proxy Manager to the `immich-proxy` instance? Have you set `https://<your-domain>:<yourport>/api` as URL in the mobile app?

**Reply by micogg** [reply]
I'm pointing NGINX Proxy Manager to the internal IP http://192.168.1.101:10323

In the app, I tried 
- https://example.duckdns.org:10323/api
- https://example.duckdns.org
The second simplified link works internally and externally via browser, just not via the Immich app.

I have attached the nginx conf as it is long.
[nginx_conf.txt](https://github.com/immich-app/immich/files/9766156/nginx_conf.txt)

**Reply by alextran1502** [reply]
I believe you should use `https://example.duckdns.org/api` since you've point your domain to the internal IP with the port, so you don't have to append the port after the domain anymore

**Reply by micogg** [reply]
Alex came through with the obvious answer, that I'm not sure why I didn't try -.- Thank you both!
